tests/foreman/cli/test_remoteexecution.py

```diff
@@ -197,3 +235,17 @@ def test_positive_run_scheduled_job_template(self):
 
         @Assert: Verify the job was successfully ran after the designated time
         """
+        system_current_time = ssh.command('date +"%b %d %Y %I:%M%p"').stdout[0]
```

Comment:
Sorry for delay, if there is no risk in this approach, I'd prefer to keep it as it is, it should work both locally and in automation.